src/rules/ban_ts_comment.rs

```diff
@@ -8,6 +8,18 @@ use swc_common::Span;
 
 pub struct BanTsComment;
 
+/*
+ * This rule differs from typescript-eslint.  In typescript-eslint the following
+ * defaults apply:
+ * ts-expect-error: allowed with comment
+ * ts-ignore: not allowed
+ * ts-nocheck: not allowed
+ *
+ * This rules defaults:
+ * ts-expect-error: allowed with comment
+ * ts-ignore: allowed with comment
+ * ts-nocheck: allowed with comment
+ */
 impl BanTsComment {
```

Comment:
```suggestion
/// This rule differs from typescript-eslint. In typescript-eslint the following
/// defaults apply:
/// - ts-expect-error: allowed with comment
/// - ts-ignore: not allowed
/// - ts-nocheck: not allowed
///
/// This rules defaults:
/// - ts-expect-error: allowed with comment
/// - ts-ignore: allowed with comment
/// - ts-nocheck: allowed with comment
pub struct BanTsComment;

impl BanTsComment {
```

If we append the comment on the `BanTsComment` struct in three slashes style, we can get documentation like this. I think it would be a little more useful for other developers 👍 

![image](https://user-images.githubusercontent.com/23649474/96326663-61030680-106d-11eb-927c-507169b4407a.png)

Reply:
Thanks for the tip!